### PR TITLE
Updated translations from Transifex

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: two_factor/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -252,44 +290,61 @@ msgid ""
...
+msgstr ""
+
+#: templates/two_factor/core/setup.html:24
+msgid "TOTP Secret:"
+msgstr ""
 
-#: templates/two_factor/core/setup.html:20
+#: templates/two_factor/core/setup.html:25
+msgid "Then, enter the token generated by the app."
+msgstr "Por favor, introduce el token generado por tu generador de tokens."
```
Comment (jpaniagualaconich) [inline two_factor/locale/es/LC_MESSAGES/django.po:321]:
```suggestion
msgstr "Luego introduzca el código generado por dicha aplicación."
```

**Review by dekkers — APPROVED** [review]

File: two_factor/locale/es/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -325,100 +380,71 @@ msgstr "Está a punto de desactivar la autenticación de dos factores. Esto debi
...
 msgid ""
 "If your primary method is not available, we are able to send backup tokens "
 "to the phone numbers listed below."
 msgstr "Si su método principal de autentificación no está disponible, podemos enviar códigos de respaldo a los números de teléfono que se enumeran a continuación."
 
-#: templates/two_factor/profile/profile.html:25
+#: templates/two_factor/profile/profile.html:20
 msgid "Are you sure?"
-msgstr ""
+msgstr "¿Estás seguro?"
```
Comment (jpaniagualaconich) [inline two_factor/locale/es/LC_MESSAGES/django.po:404]:
```suggestion
msgstr "¿Está seguro?"
```

File: two_factor/locale/es/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -325,100 +380,71 @@ msgstr "Está a punto de desactivar la autenticación de dos factores. Esto debi
...
 "backup tokens."
 msgstr "Si no tiene ningún dispositivo con usted, puede acceder a su cuenta usando códigos de respaldo."
 
-#: templates/two_factor/profile/profile.html:41
+#: templates/two_factor/profile/profile.html:36
 #, python-format
 msgid "You have only one backup token remaining."
 msgid_plural "You have %(counter)s backup tokens remaining."
 msgstr[0] "Le queda un solo código de respaldo."
 msgstr[1] "Le quedan %(counter)scódigos de respaldo."
```
Comment (jpaniagualaconich) [inline two_factor/locale/es/LC_MESSAGES/django.po:421]:
```suggestion
msgstr[1] "Le quedan %(counter)s de códigos de respaldo."
```

File: two_factor/locale/es/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -325,100 +380,71 @@ msgstr "Está a punto de desactivar la autenticación de dos factores. Esto debi
...
 msgstr[2] "Le quedan %(counter)scódigos de respaldo."
```
Comment (jpaniagualaconich) [inline two_factor/locale/es/LC_MESSAGES/django.po:422]:
```suggestion
msgstr[2] "Le quedan %(counter)s códigos de respaldo."
```

**Comment by claudep** [comment]
Thanks @jpaniagualaconich  for reviewing the translation. However, the changes must be done at the Transifex level, otherwise they will be overwritten next time we fetch translations. Could you ping me when that's done? If you are not member of the translation team of Transifex, request to join and tell me your username.

**Comment by jpaniagualaconich** [comment]
@claudep doesn't transifex let you upload the complete, updated `.po` ?

**Comment by claudep** [comment]
Ah yes, it should. Will see that tomorrow if noone else beats me...

**Comment by jpaniagualaconich** [comment]
that'd be awesome. Thanks!

**Pull request merged**